pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -504,15 +507,23 @@ func (l LibvirtWrapper) SetupLibvirt() (err error) {
 		return err
 	}
 
-	if envVarValue, ok := os.LookupEnv("LIBVIRT_DEBUG_LOGS"); ok && (envVarValue == "1") {
+	if logFilters, enableDebugLogs := getLibvirtLogFilters(); enableDebugLogs {
 		libvirdDConf, err := os.OpenFile(runtimeLibvirtdConfPath, os.O_APPEND|os.O_WRONLY, 0644)
 		if err != nil {
 			return err
 		}
 		defer util.CloseIOAndCheckErr(libvirdDConf, &err)
 
-		// see https://libvirt.org/kbase/debuglogs.html for details
-		_, err = libvirdDConf.WriteString("log_filters=\"3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 1:*\"\n")
+		var actualLogFilters string
+		if customLogFilters != nil && *customLogFilters != "" {
+			actualLogFilters = *customLogFilters
+		} else {
+			actualLogFilters = logFilters
+		}
```

Comment:
Wouldn't it make sense to pass `customLogFilters` to `getLibvirtLogFilters()` and move this logic to that function? That way, the decision on whether log filters should be enabled and what the actual setting should be would live all in one place, and `SetupLibvirt()` would simply make it happen by writing the configuration file.

Reply:
Yup, nice idea! I agree.